android/src/main/new_arch/CMakeLists.txt

```diff
@@ -43,16 +43,26 @@ target_link_libraries(
         ReactAndroid::reactnative
 )
 
-target_compile_options(
-        ${LIB_TARGET_NAME}
-        PRIVATE
-        -DLOG_TAG=\"ReactNative\"
-        -fexceptions
-        -frtti
-        -Wall
-        -std=c++20
-        -DREACT_NATIVE_MINOR_VERSION=${ReactAndroid_VERSION_MINOR}
-)
+if (ReactAndroid_VERSION_MINOR LESS_EQUAL 80)
+        target_compile_options(
+                ${LIB_TARGET_NAME}
+                PRIVATE
+                -DLOG_TAG=\"ReactNative\"
+                -fexceptions
+                -frtti
+                -Wall
+                -std=c++20
+                -DREACT_NATIVE_MINOR_VERSION=${ReactAndroid_VERSION_MINOR}
```

Comment:
Let's move `target_compile_options(... -DREACT_NATIVE_MINOR_VERSION=...)` outside of the conditional.